src/test/java/com/example/survey/SurveyApplicationTests.java

```diff
@@ -35,6 +35,8 @@
     @Autowired
     private SurveyRepo surveyRepo;
     @Autowired
+    private QuestionRepo questionRepo;
+	@Autowired
```

Comment:
nit: extra spacing

Reply:
huh, this isn't showing up in my nvim or intellJ, tried reformatting the line anyways and it didn't produce a diff to git

Reply:
just a nit, no big deal 😛 I'm fine with merging